### PR TITLE
LTEUI - GUI for LaserTracker, common EUI classes

### PR DESCRIPTION
**Pull request by pkubanek**
- StatusLabel instead of function to retrieve status string
- LaserTracker EUI
- EUIWindow, CSCControlWidget
- CompressorCSC redesigned
- M1M3 CSC based on EUIWindow
- Updated version history
- Raise RuntimeError when index isn't provided for indexed remote

### CODE REVIEW

File: python/lsst/ts/cRIOpy/GUI/SAL/SALStatusBar.py [hunk 1]
```diff
@@ -37,20 +37,17 @@ class SALStatusBar(QStatusBar):
     comms : `[SALComm]`
         SALComms for heartbeat displays. ErrorCode from the first SALComm will
         be displayed as well.
-    detailedStateFunction : `def(int)`, optional
-        If provided, a label showing detailedState (result of
+    stateLabel : `QLabel`, optional
+        If provided, aa label showing detailedState (result of
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/GUI/SAL/SALStatusBar.py:41]: aa --> a

File: python/lsst/ts/cRIOpy/LTGUI.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# -'''- coding: utf-8 -'''-
+
+# This file is part of M1M3 SS GUI.
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/LTGUI.py:3]: not M1M3

**Review by teweitsai — APPROVED** [review]
Looks good.

File: python/lsst/ts/cRIOpy/AirCompressor/CompressorsPageWidget.py [hunk 3]
```diff
@@ -53,39 +45,54 @@
...
-from ..GUI.SAL import SALCommand, SummaryStateLabel, VersionWidget
-from ..GUI.SAL.SALComm import MetaSAL, warning
+from ..GUI.SAL import CSCControlWidget, SummaryStateLabel, VersionWidget
 from ..GUI.SAL.SALLog import Widget as SALLogWidget
 
+__all__ = ["CompressorsPageWidget"]
+
+
+TEXT_POWER_ON = "Power &On"
+TEXT_POWER_OFF = "Power O&ff"
```
[the remark this answers is not in the document]
Comment (pkubanek) [reply python/lsst/ts/cRIOpy/AirCompressor/CompressorsPageWidget.py:55]: I cannot use &O for shortcut as that's already used for On

File: python/lsst/ts/cRIOpy/GUI/SAL/CSCControlWidget.py [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+    comm : `MetaSAL`
+        SAL object to receive data / run the commands.
+    buttons : `[str]`, optional
+        Text for defaults buttons.
+    extra_Buttons_commands : `dict[str, str]', optional
+        Commands for buttons texts.
+    state_signal : `str`, optional
+        Topic providing CSC state.
+    state_field : `str`, optional
+        Field inside topic providing CSC state. Defaults to
```
[the remark this answers is not in the document]
Comment (pkubanek) [reply python/lsst/ts/cRIOpy/GUI/SAL/CSCControlWidget.py:52]: Thanks. summaryState, fixed in upcoming PR

File: python/lsst/ts/cRIOpy/LTGUI.py (hunk 2 above)
Comment (pkubanek) [reply python/lsst/ts/cRIOpy/LTGUI.py:3]: Changed to EUI.

**Pull request merged**